mlflow/server/js/src/components/ExperimentView.js

```diff
@@ -28,7 +28,7 @@ import Utils from '../utils/Utils';
 export const DEFAULT_EXPANDED_VALUE = false;
 
 
-class ExperimentView extends Component {
+export class ExperimentView extends Component {
```

Comment:
Also export the unconnected component class for use in tests (see https://redux.js.org/recipes/writing-tests#connected-components)